.github/workflows/create-release.yml

```diff
@@ -12,113 +12,57 @@ jobs:
-          conda info
-
-      - name: "Install iRODS clients"
-        run: |
-          conda create -y -n "$CONDA_TEST_ENVIRONMENT"
-          conda install -y -n "$CONDA_TEST_ENVIRONMENT" "irods-icommands=${{ matrix.irods }}"
-          conda install -y -n "$CONDA_TEST_ENVIRONMENT" "baton=${{ matrix.baton }}"
-
-      - name: "Configure iRODS clients"
-        run: |
-          conda activate "$CONDA_TEST_ENVIRONMENT"
-
-          mkdir -p "$HOME/.irods"
-          cat <<'EOF' > "$HOME/.irods/irods_environment.json"
-          {
-              "irods_host": "localhost",
-              "irods_port": 1247,
-              "irods_user_name": "irods",
-              "irods_zone_name": "testZone",
-              "irods_home": "/testZone/home/irods",
-              "irods_default_resource": "replResc",
-              "irods_default_hash_scheme": "MD5"
-          }
-          EOF
-
-          echo "irods" | script -q -c "iinit" /dev/null
-          ienv
-          ils
+      - name: "Fetch Tags"
+        # Workaround for https://github.com/actions/checkout/issues/290
+        run: git fetch --tags --force
 
-          baton-do --version
-
-      - name: "Cache Go modules"
-        uses: actions/cache@v4
+      - name: "Set up Go"
+        uses: actions/setup-go@v5
         with:
-          path: |
-            "$HOME/go/pkg/mod"
-            "$HOME/.cache/go-build"
-          key: ${{ runner.os }}-key1-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-key1-go-
+          go-version-file: "go.mod"
 
-      - name: "Install test runner"
+      - name: "Get release variables"
         run: |
-          go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
-          go get github.com/onsi/gomega/...
+          echo 'RELEASE_VERSION='$(git describe --always --tags --dirty) >> $GITHUB_ENV
+          echo 'MASTER_SHA='$(git rev-parse origin/master) >> $GITHUB_ENV
+          
+          echo 'GIT_URL='$(git remote get-url origin) >> $GITHUB_ENV
+          echo 'GIT_COMMIT='$(git log --pretty=format:'%H' -n 1) >> $GITHUB_ENV
 
-      - name: "Build distribution"
+      - name: "Build executables"
         run: |
-          conda activate "$CONDA_TEST_ENVIRONMENT"
           make dist
 
-      - name: "Get release version"
-        run: |
-          git status
-          version=$(git describe --always --tags --dirty)
-          echo "$version" | grep dirty && git diff
-
-          echo VALET_VERSION="version" >> $GITHUB_ENV
-
-      - name: "Get master shasum"
-        run: |
-          echo MASTER_SHA=$(git rev-parse origin/master) >> $GITHUB_ENV
-
-      - name: "Create release"
+      - name: "Create Release"
         uses: ncipollo/release-action@v1.14.0
         with:
-          name: ${{ env.VALET_VERSION }}
+          name: ${{ env.RELEASE_VERSION }}
           prerelease: ${{ !(github.sha == env.MASTER_SHA) }}
           artifacts: "*.tar.bz2,*.tar.bz2.sha256"
```

Comment:
You also need to look for the archive inside ./build, I think the release action only looks in the cwd